Cargo.toml

```diff
@@ -164,7 +164,7 @@ cumulus-client-consensus-common = "0.15.0"
 cumulus-client-consensus-proposer = "0.14.0"
 cumulus-client-service = "0.15.0"
 
-# Paseo
-asset-hub-paseo-runtime = { git = "https://github.com/paseo-network/runtimes/", tag = "v1.2.5-system-chains", default-features = false }
-paseo-runtime = { git = "https://github.com/paseo-network/runtimes/", tag = "v1.2.5-system-chains", default-features = false }
-paseo-runtime-constants = { git = "https://github.com/paseo-network/runtimes/", tag = "v1.2.5-system-chains", default-features = false }
+# Paseo (note: using polkadot as stopgap until paseo updated to polkadot sdk v1.14.0)
+asset-hub-polkadot-runtime = { git = "https://github.com/polkadot-fellows/runtimes", rev = "4260e50", default-features = false }
```

Comment:
What specific commit is this? Why are we not pointing to e.g. a tag?

Reply:
As they dont generate a runtime release for each bump of the SDK, this commit is the one that updates the runtimes to 1.14: https://github.com/polkadot-fellows/runtimes/commits/main/

@chungquantin could you try it without the commit to see if it all still compiles? It seems the fellowship runtimes are still on 1.14 so the latest main should suffice, relying on the lock file to pin.